verification/curator-service/ui/src/components/EpidNavbar.tsx

```diff
@@ -1,52 +1,84 @@
 import { Link } from "react-router-dom";
+import React from "react";
 import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
+import { WithStyles } from "@material-ui/core/styles/withStyles";
+import axios from 'axios';
 
-const useStyles = makeStyles((theme: Theme) =>
-    createStyles({
-        root: {
-            flexGrow: 1,
-        },
-        menuButton: {
-            marginRight: theme.spacing(2),
-        },
-        title: {
-            flexGrow: 1,
-        },
-    }),
-);
-
-export default function EpidNavbar() {
-    const classes = useStyles();
-
-    useEffect(() => {
-        // TODO: Fetch /api/profile and display user info in navbar.
-    });
-
-    return (
-        <div className={classes.root}>
-            <AppBar position="static">
-                <Toolbar>
-                    <Link to="/">
-                        <IconButton
-                            data-testid="home-btn"
-                            edge="start"
-                            className={classes.menuButton}
-                            aria-label="menu">
-                            <HomeIcon />
-                        </IconButton>
-                    </Link>
-                    <Typography variant="h6" className={classes.title}>
-                        epid
+const styles = (theme: Theme) => createStyles({
+    root: {
+        flexGrow: 1,
+    },
+    menuButton: {
+        marginRight: theme.spacing(2),
+    },
+    title: {
+        flexGrow: 1,
+    },
+});
+
+interface User {
+    email: string;
+}
+
+// Cf. https://material-ui.com/guides/typescript/#augmenting-your-props-using-withstyles
+interface Props extends WithStyles<typeof styles> { }
+
+class EpidNavbar extends React.Component<Props, User> {
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            email: "",
+        };
+    }
+
+    componentDidMount() {
+        axios.get<User>('/auth/profile')
+            .then((resp) => {
+                this.setState({ email: resp.data.email });
+            }).catch((e) => {
+                this.setState({ email: "" });
+                console.error(e);
+            });
+    }
+
+    render() {
+        const { classes } = this.props;
+        return (
+            <div className={classes.root} >
+                <AppBar position="static">
+                    <Toolbar>
+                        <Link to="/">
+                            <IconButton
+                                data-testid="home-btn"
+                                edge="start"
+                                className={classes.menuButton}
+                                aria-label="menu">
+                                <HomeIcon />
+                            </IconButton>
+                        </Link>
+                        <Typography variant="h6" className={classes.title}>
+                            epid
                     </Typography>
```

Comment:
super-nit: increase indentation to match <Typography

Reply:
thanks, weird I have autoformat on save but it didn't catch that. Done.